_config.yml

```diff
@@ -12,6 +12,10 @@ collections:
     title: All Articles
     output: true
     permalink: "/:path/articles/"
+  series:
+    title: Series
+    output: true
+    permaline: "/:path/series"
```

Comment:
Check this line for spelling. It should also end in a trailing slash.

When you submit PRs for new collections, you should make sure that the sample.md file is being properly created in the `_site` folder.

Reply:
Updated for spelling & trailing slash. 

I keep getting this error whenever I push changes to my branches:
`hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.`

I have to go into the file & edit in in-line for the edit. I have been rebasing before editing - not sure why this would be an issue?